services/director-v2/src/simcore_service_director_v2/modules/dask_clients_pool.py

```diff
@@ -80,7 +80,7 @@ async def _concurently_safe_acquire_client() -> DaskClient:
                         tasks_file_link_type = (
                             self.settings.COMPUTATIONAL_BACKEND_DEFAULT_CLUSTER_FILE_LINK_TYPE
                         )
-                    if cluster.type is ClusterTypeInModel.ON_DEMAND:
+                    if cluster.type == ClusterTypeInModel.ON_DEMAND.value:
```

Comment:
Do you really need `.value` here? Note that both are `ClusterTypeInModel` and inherits from both str and Enum.

Check https://github.com/pcrespov/sandbox-python/blob/python-stuff/test_enumerations.py

Reply:
so this stem from the dask-gateways and I do not want to touch that now as I'm not sure what this will trigger. therefore I'll keep it as is.